test/Npgsql.Tests/TransactionTests.cs

```diff
@@ -37,17 +34,19 @@ public async Task Rollback([Values(PrepareOrNot.NotPrepared, PrepareOrNot.Prepar
             await using var conn = await OpenConnectionAsync();
             await using var _ = await CreateTempTable(conn, "name TEXT", out var table);
 
-            var tx = conn.BeginTransaction();
-            var cmd = new NpgsqlCommand($"INSERT INTO {table} (name) VALUES ('X')", conn, tx);
-            if (prepare == PrepareOrNot.Prepared)
-                cmd.Prepare();
-            cmd.ExecuteNonQuery();
-            Assert.That(conn.ExecuteScalar($"SELECT COUNT(*) FROM {table}"), Is.EqualTo(1));
-            tx.Rollback();
-            Assert.That(tx.IsCompleted);
-            Assert.That(await conn.ExecuteScalarAsync($"SELECT COUNT(*) FROM {table}"), Is.EqualTo(0));
-            Assert.That(() => tx.Connection, Throws.Exception.TypeOf<InvalidOperationException>());
-            await tx.DisposeAsync();
+            var tx = await conn.BeginTransactionAsync();
+            await using (var disposedTx = tx)
```

Comment:
Since `disposedTx` is never used, you can use a discard (underscore). Though I'm not sure switching to using actually improves anything for this test.

Reply:
I did try it, but it doesn't compile 😢

Reply:
```suggestion
            await using (tx)
```

Reply:
This works for me